src/sensitivities/indexing.jl

```diff
@@ -5,7 +5,11 @@ for i = 1:7
     @eval @explicit_intercepts getindex $T [[true]; fill(false, $i - 1)]
 end
 
-∇(Ā, ::typeof(getindex), ::Type{Arg{1}}, p, y, ȳ, A, inds...) = setindex!(Ā, ȳ, inds...)
+# ∇(Ā, ::typeof(getindex), ::Type{Arg{1}}, p, y, ȳ, A, inds...) = setindex!(Ā, ȳ, inds...)
```

Comment:
Perhaps just remove the line as opposed to commenting it out?

Reply:
Ah, yeah, good point.